bookwyrm/views/books/editions.py

```diff
@@ -103,4 +103,18 @@ def switch_edition(request):
         readthrough.book = new_edition
         readthrough.save()
 
+    ratings = models.ReviewRating.objects.filter(
+        book__parent_work=new_edition.parent_work, user=request.user
+    )
+    for rating in ratings.all():
+        rating.book = new_edition
+        rating.save()
```

Comment:
If the above above is correct, perhaps it means this block could be completely dropped. I'm sure one of the maintainers will be able to confirm.

Thanks again!

Reply:
Yes, this is it! I confirmed this by creating a ReviewRating and see that the Review loop picks it up. It makes sense. I will update the PR now. Thank you!